src/tools/cephfs/cephfs-shell

```diff
@@ -552,11 +550,11 @@ class CephFSShell(Cmd):
                         o_mode |= t_mode
 
             if o_mode < 0:
-                parser.error("invalid mode: %s\n"
-                             "mode cannot be negative" % values)
-            if o_mode > 0o777:
-                parser.error("invalid mode: %s\n"
-                             "mode cannot be greater than octal 0777" % values)
+                parser.error(f"invalid mode: {values}\n"
+                             "mode cannot be negative")
+            if o_mode > 0o7777:
+                parser.error(f"invalid mode: {values}\n"
+                             "mode cannot be greater than octal 0777")
```

Comment:
The error message needs to be updated here.

Reply:
Done!!